src/test/java/com/google/api/codegen/testdata/csharp_gapic_snippets_library.baseline

```diff
@@ -0,0 +1,696 @@
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+using Google.Api.Gax;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+using Grpc.Core;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Threading;
+using System.Threading.Tasks;
+using UpdateBookIndexRequest.Types;
+
+namespace Google.Example.Library.V1.Snippets
+{
+    public class LibraryServiceClient
+    {
+        public async Task CreateShelfAsync()
+        {
+            // Snippet: CreateShelfAsync(Shelf,CallSettings)
+            // Additional: CreateShelfAsync(Shelf,CancellationToken)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            Shelf shelf = new Shelf();
+            // Make the request
+            Shelf response = await libraryServiceClient.CreateShelfAsync(shelf);
+            // End snippet
+        }
+
+        public void CreateShelf()
+        {
+            // Snippet: CreateShelf(Shelf,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            Shelf shelf = new Shelf();
+            // Make the request
+            Shelf response = libraryServiceClient.CreateShelf(shelf);
+            // End snippet
+        }
+
+        public async Task GetShelfAsync()
+        {
+            // Snippet: GetShelfAsync(string,CallSettings)
+            // Additional: GetShelfAsync(string,CancellationToken)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            string formattedName = LibraryServiceClient.FormatShelfName("[SHELF]");
+            // Make the request
+            Shelf response = await libraryServiceClient.GetShelfAsync(formattedName);
+            // End snippet
+        }
+
+        public void GetShelf()
+        {
+            // Snippet: GetShelf(string,CallSettings)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
+            string formattedName = LibraryServiceClient.FormatShelfName("[SHELF]");
+            // Make the request
+            Shelf response = libraryServiceClient.GetShelf(formattedName);
+            // End snippet
+        }
+
+        public async Task GetShelfAsync()
+        {
+            // Snippet: GetShelfAsync(string,SomeMessage,CallSettings)
+            // Additional: GetShelfAsync(string,SomeMessage,CancellationToken)
+            // Create client
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+            // Initialize request argument(s)
```

Comment:
Nit for the future... we know how many arguments there are, so we don't need the (s).